less/mosaico/header.less

```diff
@@ -0,0 +1,67 @@
+.mosaico {
+    .header {
+        grid-area: head;
+        display: grid;
+
+        grid-template-rows: 0.1fr 65px 0.1fr;
+        grid-template-columns: 1fr 1fr 1fr;
+
+        font: 1000 12px 'Roboto', sans-serif;
+
+        .left-links {
+            grid-row-start: 1;
+            grid-column: 1 / span 2;
+        }
+
+        .right-links {
+            grid-row: 1 / span 2;
+            grid-column-start: 3;
+
+            color: #00a1ab;
+
+            ul {
+                margin-top: 0px;
+                list-style-type: none;
+                text-align: right;
+            }
+
+            .prenumerera-link {
+                border-radius: 3px;
+                background-color: #00a1ab;
+                color: #fff;
+                text-align: center;
+                padding: 3px 3px 1px 3px;
+            }
+        }
+
+        .logo {
```

Comment:
I'd still argue the use of scoping is unnecessary here. 

Consider a css class
```css
.mosaico {
  h2 {
    color: red;
  } 
}
```
This is pretty obvious what it does. We know there's a default styling for `h2` defined by the browser, but here we explicitly say "hey, we have this known element `h2` in the global scope, but under `mosaico` it should not follow the normal behaviour, but instead it should be red."

If we aren't going to have e.g. `logo` anywhere else in the document, then nesting it is not needed. However, if we have a generic `logo` somewhere, then having additional rules based on scope makes sense. Here we have a base rule for `logo` but change its color depending on where we are using it.

```css
.logo {
  width: 100px;
  height: 100px;
  color: blue;
}
.mosaico {
  .logo {
    color: red;
  }
}
.vetrina {
  .logo {
    color: green;
  }
}
```
Also if `logo` exists only in a specific scope, it's hard to tell which one by reading the DOM.
```html
<div class="a">
  <div class="b">
    <div class="c">
      <div class="d">
        <div class="logo">
```
```css
.b {
  .logo {
    color: yellow;
  }
}
```
If we required `logo` to be a direct child of a class, meaning classes `a`, `b`, `c` and `d` having their own definitions for it, we'd end up in a nightmare.

Reply:
Yes this makes sense. 

But I'd like to clarify the BEM syntax above all so that I can make less output it. I'd like to understand if we're following the standard or not. It doesn't seem so, which is fine, but then what are the rules? 

When they are defined, there should be a README somewhere with them written in.

Reply:
I'd still say that we should follow an official standard as it is probably easier for everyone, unless we have specific reasons not to do so.

Reply:
@ae-mo @vapaj If we do decide to follow a particular BEM standard, I think [CSS Tricks's BEM 101](https://css-tricks.com/bem-101/) article has just the right amount of BEM tips and recommended syntax for us to follow easily, and it's just a 5-minute read. I know other sources go into more depth on BEM, but I think this article hits that nice 80/20 (benefit/effort) sweet spot. I found it an excellent refresher on BEM myself.